hash/sms.xml

```diff
@@ -8091,7 +8091,9 @@ license:CC0-1.0
 		<description>Wing Warriors (fix)</description>
 		<year>2024</year>
 		<publisher>EGS</publisher>
-		<info name="language" value="English/French/Portuguese (Brazil)/Spanish" />
+		<info name="language" value="English" />
+		<info name="language" value="French" />
+		<info name="language" value="Portuguese (Brazil)/Spanish" />
```

Comment:
More missing `Portuguese (Brazil)/Spanish` splits. Again multiple times in this file.